### PR TITLE
REACH-164 Widget for String node in customizer

### PR DESCRIPTION
**Pull request by pbidenko**
Widget for String node

### CODE REVIEW

File: app/app.html [hunk 1]
```diff
@@ -471,6 +471,26 @@
         <% }); %>
     </ul>
 
+    <div class="dropdown keep-open node-settings" style="position:absolute">
```
Comment (pbidenko) [inline app/app.html]: Added "Name" field and "Hide in customizer" checkbox for string node.

File: app/customizer.html [hunk 2]
```diff
@@ -297,6 +297,20 @@
 
       </script>
 
+      <script type="text/template" id="widget-string-template">
```
Comment (pbidenko) [inline app/customizer.html]: Template for string node widget.

File: app/scripts/views/customizer/Widgets/String.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
+define(['backbone', 'BaseWidgetView'], function(Backbone, BaseWidgetView) {
```
Comment (pbidenko) [inline app/scripts/views/customizer/Widgets/String.js]: Widget for string node.

**Comment by pboyer** [comment]
Thanks @pbidenko!  I fixed the merge conflict, as well.  Nice work!

**Comment by pboyer** [comment]
Github is being weird about the merge, so I have to close this.  But it's in!

**Pull request closed without merging**